website/pages/en/quick-start.mdx

```diff
@@ -2,24 +2,19 @@
 title: Quick Start
 ---
 
-This guide will quickly take you through how to initialize, create, and deploy your subgraph to Subgraph Studio.
+This guide walks you through the basics of publishing and querying a subgraph on The Graph easily.
```

Comment:
I like the feeling of setting expectations that "will" brings, but I do like "walk through" over "take through". How about:
```suggestion
This guide will walk you through the basics of publishing and querying a subgraph on The Graph.
```